examples/tutorials/alpha-zero-tic-tac-toe/README.md

```diff
@@ -0,0 +1,17 @@
+# Alpha Zero Tic-Tac-Toe
```

Comment:
What are we trying to showcase with this tutorial?

Tutorials should not be an collection of potentially useful scripts, they should be specific curated set of guides to show specific things that can be done in Determined.

Reply:
Originally this was trying to show how to make a minimal change for core-api to a fairly complex model while utilizing other information sources like user data or searcher length. Would you prefer this README be better documented, the directory be moved up to examples, or the PR be closed out altogether?

Reply:
I guess my real intention is to protect our examples from ad-hoc growth.  We have a _lot_ of examples, but they're not very well curated right now.  It's kinda hard to maintain all of them as it is (we had 36 or 37 last I counted).

So I'm leaning in the direction of "let's just close this PR, unless it's serving a particular request I don't know about" (which is why I was interested in the motivation of the PR).

FWIW, we do already have a blog post with instructions for migrating a TIMM model [here](https://www.determined.ai/blog/training-arbitrary-models-core-api-determined) which serves a similar purpose I think.

Reply:
Hey, @erikwilson, I found out that we actually have a [model zoo repo](https://github.com/determined-ai/model-zoo) for projects that this would be well-suited for.  You might consider adding it there.